conftest.py

```diff
@@ -34,6 +34,18 @@ def __str__(self):
 def pytest_addoption(parser):
     # Create the cfme option group for use in other plugins
     parser.getgroup('cfme', 'cfme: options related to cfme/miq appliances')
+    # Keeping due to compatibility
+    parser.addoption("--no-tracer", dest="no_tracer", action="store_true", default=False,
+                     help="Disable the function tracer (REMOVED, doesn't do anything!)")
+    parser.addoption("--tracer", dest="tracer", action="store_true", default=False,
+                     help="Enable the function tracer (REMOVED, doesn't do anything!)")
+
+
+def pytest_cmdline_main(config):
+    for opt in ['--tracer', '--no-tracer']:
```

Comment:
how about using config.option.tracer or config.option.no_tracer

alternatively i think you can use the combination of meta and using the same destination to just use a single `deprecated_tracer_arg_used` option